src/software/SfM/main_openMVG2openMVS.cpp

```diff
@@ -107,11 +107,20 @@ bool exportToOpenMVS(
         ReadImage(srcImage.c_str(), &imageRGB);
         UndistortImage(imageRGB, cam, imageRGB_ud, BLACK);
         WriteImage(image.name.c_str(), imageRGB_ud);
+        // undistort mask and save it
+        auto maskFileName = srcImage + ".mask.png";
```

Comment:
`const std::string`
Since the type is known and will not change I would prefer to use std::string there.

Reply:
`".mask.png"` can be saved in a string and reused.
`const std::string openmvs_mask_postfix = ".mask.png"`;